### ISSUE TITLE
Roundtrip should not prorpose changes

### ISSUE THREAD

**Issue by masseelch** [ariga/atlas#392]
I ran `atlas schema inspect -d "<dsn>" > atlas.hcl`, immediately followed by `atlas schema apply -d "<dsn>" -f atlas.hcl` and the tool wanted to run some changes on my DB.

Example of the generated SQL commands:

```
-- Modify "jobs" table
ALTER TABLE `mipo`.`jobs` DROP INDEX `info`
-- Modify "jobs" table
ALTER TABLE `mipo`.`jobs` ADD INDEX `info` (`info`)
```

MySQL 5.7

**Comment by rotemtam** [comment]
@masseelch,

What's the status of this issue? Still relevant or fixed with recent improvements to the engine?

**Comment by masseelch** [comment]
Still relevant. There are a bunch of changes proposed by the engine that essentially do nothing. Looks like they only affect boolean types (`tinyint`).

**Comment by masseelch** [comment]
Fixed in #493
